### DISCUSSION TITLE
深渊记录查询bug

### DISCUSSION THREAD

**Discussion by azurstar** [Arondight/Adachi-BOT#471] [Q&A]
深渊记录只能查询一次  
如图：
![截图](https://gchat.qpic.cn/gchatpic_new/1401068666/544346435-2818798459-3C7D2804DB455CAC1E0E81BC28539A38/0?term=2)

**Comment by Arondight** [comment]
更一下代码，应该好了

**Reply by azurstar** [reply]
> 更一下代码，应该好了

已经没问题了
